examples/frames.rs

```diff
@@ -0,0 +1,18 @@
+extern crate mp3;
+
+use mp3::frame::FrameReader;
+
+use std::fs::File;
+use std::io::Read;
+
+fn main() {
+    let path = std::env::args().nth(1).unwrap();
+    let mut f = File::open(path).unwrap();
+    let mut buf = Vec::new();
+    f.read_to_end(&mut buf).unwrap();
+    let slice = mp3::trim_data(&buf).unwrap();
+    let mut frame_reader = FrameReader::new(slice);
```

Comment:
We should probably start thinking, at least from a high-level, about the API we want to expose. Might be worth opening an issue to discuss.

Reply:
Done: #9